com.chartboost.mediation/Runtime/Utilities/ChartboostMediationConverters.cs

```diff
@@ -0,0 +1,52 @@
+using Chartboost.Platforms;
+#if UNITY_ANDROID
+using Chartboost.Platforms.Android;
+#elif UNITY_IPHONE
+using Chartboost.Platforms.IOS;
+#endif
+using UnityEngine;
+
+namespace Chartboost.Utilities
+{
+    public static class ChartboostMediationConverters
+    {
+        private static float? _scaleFactor = null;
+        
+        public static float NativeToPixels(float native)
+        {
+            return native * ScaleFactor;
+        }
+        
+        public static float PixelsToNative(float pixels)
+        {
+            return pixels / ScaleFactor;
+        }
+
+        public static Vector2 NativeToPixels(Vector2 native)
+        {
+            return new Vector2(NativeToPixels(native.x), NativeToPixels(native.y));
+        }
+        
+        public static Vector2 PixelsToNative(Vector2 pixels)
+        {
+            return new Vector2(PixelsToNative(pixels.x), NativeToPixels(pixels.y));
+        }
+
+        private static float ScaleFactor
+        {
+            get
+            {
+                #if UNITY_EDITOR
+                return _scaleFactor ??= 2.5f;
```

Comment:
What do we mean by 2.5f? Pixels, factor?

Reply:
it is a multiplication factor that is applied to native units (dips in Android and points in iOS) to get the value in pixels.  
I'll have this come from a  constant